uaclient/defaults.py

```diff
@@ -83,3 +83,12 @@
     "/etc/apparmor.d/ubuntu_pro_apt_news",
     "/etc/apparmor.d/ubuntu_pro_esm_cache",
 ]
+
+CODENAME_TO_RELEASE = {
```

Comment:
Eeeehhhh I don't like how this will be maintained...
but I see this as the only reasonable way of accepting my comment about the string format :| so I can live with it.

I see Mantic on the list. I don't expect to have mantic-only contracts. Do you think this map will be used for anything non-LTS related? Otherwise, keep it LTS I guess

Reply:
Yes, I discussed with Grant and we both agree the maintenance of this is not ideal but to get the strings for the messages this seemed like one way.
To add to your comments above, if we want to standardize the strings then this would just become a dict with both the release number and the series full name.  (Also, this should be LTS only so will remove mantic)

Reply:
Now that I look at it, we can use the `system.get_distro_info` function instead of having to maintain this string?

Reply:
A less maintainable way to do this would be something like the function `system.get_distro_info` but use the distro information CSV to get the release, series name and the full name. This would be easier to use and would not have the same maintenance issue as this dict. Would just be an added function to system.

Let me know if you think thats a good idea @orndorffgrant @renanrodrigo @lucasmoura

Reply:
I think it's very possible that it could get reused in the future for something else without looking to closely. I think we might as well maintain it for our full set of supported releases

Reply:
Whoops I posted this reply without seeing anything other than the initial comment and we talked about this out of band - should have refreshed before posting :upside_down_face: